migrations/20170310165323-add-notes-to-donations-migration.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+
+    return queryInterface.addColumn('Donations', 'notes', {
```

Comment:
There was only a title field on description, so if someone wanted to add more explanation, we have no way of storing that info. This allows for more details to be attached to a donation. When displaying transaction, we can pass that info on.